Jenkinsfile_nightly

```diff
@@ -1,17 +1,33 @@
 #!groovy
 
 properties([
-  pipelineTriggers([cron('20 21 * * *')])
+  pipelineTriggers([cron('20 21 * * *')]),
+  parameters([
+    string(name: 'URL_FOR_FULL_FUNCTIONAL_TEST', defaultValue: 'https://www-ccd.aat.platform.hmcts.net',
```

Comment:
i think it should be xui now:
https://manage-case.aat.platform.hmcts.net

Reply:
Will change this to point to XUI